docs/migration/dapp_migration.md

```diff
@@ -28,8 +28,8 @@ For starters, here's a sampling of exciting perks dapps with get with the Nitro
 - **Gas Accounting**: it is now consistent with the L1 EVM, any hard-coded gas values should be changed accordingly (the same applies to any gas amount used in conjuntion with `gasleft`).
 - **No more storage gas**: there is no more concept of a separate pool of storage gas, opcodes are prices identically to the L1 EVM.
 - **Retryable Tickets**: 
-    - The submission cost is now collected/enforced in the L1 inbox and checked against the L1 transaction's `msg.value`; contracts cannot/should not rely on funds pooled in the L2 destination to cover this cost.
-    - For the redemption of retryable tickets, the calculation of the L2 transaction ID is changed, as has the transaction lifecycle of attempting multiple redemptions (i.e., after failed attempts). See [arbitrum-sdk](https://github.com/OffchainLabs/arbitrum-sdk/blob/fe3c3ee90a2d713955988dcb6a9f87732b7dbedc/src/lib/message/L1ToL2Message.ts#L547) for the new client-side flow (detailed documentation coming soon) 
+    - The submission cost is now enforced in the L1 inbox and checked against the L1 transaction's `msg.value`; contracts shouldn't rely on funds pooled in the L2 destination to cover this cost.
+    - For the redemption of retryable tickets, the calculation of the L2 transaction ID changed, as has the transaction lifecycle of attempting multiple redemptions (i.e., after failed attempts). See [arbitrum-sdk](https://github.com/OffchainLabs/arbitrum-sdk) for a reference implementation on the new client-side flow. 
```

Comment:
I'm not sure what "as has the transaction lifecycle of attempting multiple redemptions (i.e., after failed attempts)" is communicating. Maybe "and differs between different redeem attempts (i.e. after failed attempts)"?